dev-docs/modules/userId.md

```diff
@@ -220,6 +220,67 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+### Deepintent DPES ID by Deepintent
+The Deepintent DPES ID or Healthcare Identity is a shared, neutral identifier that publishers and ad tech platforms can use to recognise users even in environments where 3rd party cookies are not available. The Deepintent DPES ID is designed to respect users' privacy choices and publishers’ preferences throughout the advertising value chain. 
```

Comment:
Can you add some language about how Deepintent respects users privacy choices, do you have a link to an optout/in page?  If a user wants to express a privacy preference how can they do that?

Reply:
I will add in our opt out page here and replace the wording thanks !!

Reply:
Added the information